local-cli/runIOS/runIOS.js

```diff
@@ -52,10 +52,10 @@ function _runIOS(argv, config, resolve, reject) {
     throw new Error(`Cound't find ${args.simulator} simulator`);
   }
 
-  const simulatorFullName = `${selectedSimulator.name} (${selectedSimulator.version})`;
+  const simulatorFullName = `${selectedSimulator.name} (${selectedSimulator.version}) [${selectedSimulator.udid}]`;
```

Comment:
@grabbou I was going to do that, but I was thinking about leaving it in there to make sure people can identify exactly what simulator is launched. Just incase you have, let's say, two iPhone 6 (iOS 9.2) setup.

Thoughts?